python-backend/app/config.py

```diff
@@ -26,6 +26,7 @@ class Config(object):
 
     # Microservice URLs
     DOCUMENT_MS_URL = os.environ.get('DOCUMENT_MS_URL', 'http://localhost:5000')
+    DOCUMENT_MANAGER_URL = os.environ.get('DOCUMENT_MS_URL', 'http://localhost:5000')
```

Comment:
environment variable looks like it should be Document_Manager_URL, to be different. 
MS was supposed to mean microservice, maybe we should change that standard of follow it? 
maybe, Document_Manager_MS_URL

Reply:
done